contracts/test/mocks/MockERC1271Wallet.sol

```diff
@@ -37,10 +40,19 @@ contract MockERC1271Wallet is ISetAllowance, IERC1271Wallet {
     }
 
     function isValidSignature(bytes calldata _data, bytes calldata _signature) external view override returns (bytes4 magicValue) {
+        require(operator == _ecrecover(keccak256(_data), _signature), "MockERC1271Wallet: invalid signature");
         return ERC1271_MAGICVALUE;
     }
 
     function isValidSignature(bytes32 _hash, bytes calldata _signature) external view override returns (bytes4 magicValue) {
+        require(operator == _ecrecover(_hash, _signature), "MockERC1271Wallet: invalid signature");
         return ERC1271_MAGICVALUE_BYTES32;
     }
+
+    function _ecrecover(bytes32 hash, bytes memory signature) internal pure returns (address) {
```

Comment:
How about using OZ lib for this?
https://docs.openzeppelin.com/contracts/3.x/api/cryptography#ECDSA-recover-bytes32-bytes-

Reply:
Updated in c369e28

Reply:
Replace `_ecrecover` with `ECDSA.recover` in 59fc2da